packages/core/src/globals.ts

```diff
@@ -0,0 +1,22 @@
+import type { Scope } from '@sentry/types';
+
+interface GlobalData {
+  globalScope?: Scope;
```

Comment:
m: we need to be careful for `global{s}.ts` because of https://github.com/getsentry/sentry-javascript/pull/5969

Let's rename this file to `globalscope.ts`? Also why do we have the indirection to `GLOBAL_DATA.globalScope`, can't we point to `globalScope` directly? If we refactor to need more `GLOBAL_DATA`, items, we can add it back.